webapi/src/main/scala/org/knora/webapi/messages/admin/responder/usersmessages/UsersMessagesADM.scala

```diff
@@ -624,28 +645,26 @@ case class UserADM(id: IRI,
       }
       case _ => throw BadRequestException(s"The requested userTemplateType: $userTemplateType is invalid.")
     }
-  }
 
   /**
-    * Given an identifier, returns true if it is the same user, and false if not.
-    */
+   * Given an identifier, returns true if it is the same user, and false if not.
+   */
   def isSelf(identifier: UserIdentifierADM): Boolean = {
 
-    val iriEquals = identifier.toIriOption.contains(id)
-    val emailEquals = identifier.toEmailOption.contains(email)
+    val iriEquals      = identifier.toIriOption.contains(id)
+    val emailEquals    = identifier.toEmailOption.contains(email)
     val usernameEquals = identifier.toUsernameOption.contains(username)
```

Comment:
Although most of re-formatting done in the file makes sense, the one marked above I don't personally like and it's kind of the same "alignment idea" to arguments formatted here before. I opt to keep fields as it was before :)

Reply:
OK, I suggest that we do the reformatting in a separate PR and then reformat everything (we already tried that but then the tests didn't run through - so we need to be careful with this). Besides, I don't know how to re-do this now... (as it was done automatically by saving the file)

Reply:
It's matter of the formatter configuration. It should be rather easy to revert unwanted things.